README.md

```diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
 ```
 $ git clone https://github.com/embercsi/ember-csi-kubevirt.git
 $ cd ember-csi-kubevirt/
-$ make cluster-up
-$ make deploy
-$ make demo
+$ make all
+```
+
+At this point we have  fully working cluster with KubeVirt and Ember-CSI deployed. We can now deploy VMs.
+
+```
+# Creating a virtual machine
+$ kubectl apply -f https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml
+
+# After deployment you can manage VMs using the usual verbs:
+$ kubectl get vms
+$ kubectl get vms -o yaml testvm
+
+# To start a VM you can use
+$ ./virtctl start testvm
```

Comment:
Should be ./cluster/virtctl.sh - "make all" was executed in the top level dir, so we need the leading prefix dir here and for all other commands

Reply:
again, we will use the alias 'v' here.